src/core/qgsapplication.h

```diff
@@ -405,6 +405,12 @@ class CORE_EXPORT QgsApplication : public QApplication
     //! Returns the path with utility executables (help viewer, crssync, ...)
     static QString libexecPath();
 
+    /**
+     * Returns the path where QML components are installed for QGIS Quick library
```

Comment:
Can you document what is returned if it's built without quick support?

Reply:
It returns CMake's (DEFAULT_)QML_SUBDIR regardless of if there is something in the directory (==WITH_QUICK) or not. Would it make sense to add here something like: "Could be empty" or so?

Reply:
I was referring to this https://github.com/PeterPetrik/QGIS/blob/qgsquick_1_canvas/CMakeLists.txt#L668

Reply:
fixed, returns empty QString and docs improved